cypress/support/API/addEmpAPI/addEmpAPI.ts

```diff
@@ -0,0 +1,53 @@
+let empNumber: number;
+export default class employee {
+  addEmloyeeViaAPI(
+    firstName: string,
+    middleName: string,
+    lastName: string,
+    empPicture: null,
+    employeeId: string,
+    password: string
+  ) {
+    return cy
+      .request({
+        method: "POST",
+        url: "/api/v2/pim/employees",
+        body: {
+          firstName: firstName,
+          middleName: middleName,
+          lastName: lastName,
+          empPicture: empPicture,
+          employeeId: employeeId,
+        },
+      })
+      .then((response) => {
+        expect(response).property("status").to.equal(200);
+        console.log(response, "emp");
+        empNumber = response.body.data.empNumber;
+        cy.request({
+          method: "POST",
+          url: "/api/v2/admin/users",
+          body: {
+            username: firstName,
+            password: password,
+            status: true,
+            userRoleId: 2,
+            empNumber: empNumber,
+          },
+        });
+      });
+  }
+
+  deleteEmployee() {
+    cy.request({
+      // delete the created employee
+      method: "DELETE",
+      url: "/api/v2/pim/employees",
+      body: {
+        ids: [empNumber],
+      },
+    }).then((response) => {
+      expect(response).property("status").to.equal(200);
```

Comment:
No need for this assertion since you not check the api here , you use it (Not API Testing)